cli/commands/play.py

```diff
@@ -51,40 +55,51 @@ def play(
     play a track matching <keyword>.
     """
     from cli.commands.shuffle import shuffle as shuffle_cmd
-
+    
     requests = []
     keyword = ' '.join(keyword)
+    
     if keyword:
-        import urllib.parse as ul
-        pager = Spotify.Pager(
-            'search',
-            params={
-                'q': ul.quote_plus(keyword),
-                'type': play_type,
-            },
-            content_callback=lambda c: c[play_type+'s'],
-        )
-        if len(pager.content['items']) == 0:
-            click.echo('No results found for "{}"'.format(keyword), err=True)
-            return
-
-        item = pager.content['items'][0]
-        if play_type == 'track':
-            kwargs['data'] = {
-                'context_uri': item['album']['uri'],
-                'offset': {
-                    'uri': item['uri'],
+        if play_type == 'uri':            
```

Comment:
looks like this implementation functions as expected. good one!

i think this flow here would look better and produce a nicer diff if it were structured like:
```py
if play_type == 'uri':
    # do your new stuff here
else if keyword:
    # the original keyword handling code
```

Reply:
Much better!